mitosheet/src/mito/utils/actions.tsx

```diff
@@ -629,6 +635,168 @@ export const getActions = (
             searchTerms: ['format', 'decimals', 'percent', '%', 'scientific', 'Mill', 'Bill', 'round'],
             tooltip: "Format all of the selected columns as percents, choose the number of decimals, etc. This only changes the display of the data, and does not effect the underlying dataframe."
         },
+        [ActionEnum.Formulas_Dropdown_Math]: {
+            type: 'build-time',
+            staticType: ActionEnum.Formulas_Dropdown_Math,
+            icon: MathFunctionsIcon,
+            longTitle: 'Math Formulas',
+            toolbarTitle: 'Math & Trig',
+            actionFunction: () => {
+                // We turn off editing mode, if it is on
+                setEditorState(undefined);
```

Comment:
I think we _don't_ want to do this in these functions. We want the user to be able to click these dropdowns if they have an open cell editor, methinks....